src/Components/Login/login.module.css

```diff
@@ -0,0 +1,53 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-self: center;
+  width: 40%;
+  height: 30rem;
+  margin-top: 8rem;
+  margin-left: 20%;
+  margin-bottom: 8rem;
+  border-radius: 5px;
+  box-shadow: 0px 9px 46px 8px rgba(0, 0, 0, 0.12), 0px 24px 38px 3px rgba(0, 0, 0, 0.14);
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  background-color: antiquewhite;
+}
+
+.emailContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-start;
+  align-content: center;
+  width: 55%;
+  height: 6rem;
+}
+
+.passwordContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  align-content: center;
+  width: 55%;
+  height: 5rem;
+}
+
+.eyeButton {
+  display: flex;
+  align-self: center;
+  border: none;
+  background-color: transparent;
+  scale: 1.7;
```

Comment:
I would leave the eye with its original size to mark consistency with the others used in admin and member